app/src/features/projects/CreateProjectPage.test.tsx

```diff
@@ -30,19 +28,29 @@ const renderContainer = () => {
 describe('CreateProjectPage', () => {
   beforeEach(() => {
     // clear mocks before each test
-    mockBiohubApi().createProject.mockClear();
     mockBiohubApi().getAllCodes.mockClear();
   });
 
   afterEach(() => {
     cleanup();
   });
 
-  it('shows the page title', async () => {
+  it('renders the initial default page correctly', async () => {
```

Comment:
can we add a test case for the partial vs non partial form? to test that that functionality works as expected and that the snaps look appropriate? :)